package.json

```diff
@@ -30,6 +30,7 @@
     "react-native-reanimated": "^2.2.0",
     "react-native-safe-area-context": "^3.2.0",
     "react-native-screens": "^3.3.0",
+    "react-native-splash-screen": "^3.2.0",
```

Comment:
I think this package has not been updated for almost 3 years. Don't we have any other alternatives?

Reply:
All the other thins are fine I guess and my only concern is about the package maintenance of the original developer(s).

Reply:
@shehand Considering what you have said above, I have replaced the package `react-native-splash-screen` with `react-native-bootsplash` which seems more updated and promising to me! Please verify.